config/management/README.md

```diff
@@ -67,3 +67,74 @@ While this is compiled as a single binary it provides several different faciliti
   storage -- leveraging the identity tool.
 * Converting a genesis configuration and a secure storage into a genesis.blob /
   genesis waypoint.
+
+## The Process
+
+* The association first sets up a secure storage, in this case, say GitHub. All
+  participants must have commit and read access to this repository. The
+  association will share out the GitHub repository and a distinct name for each
+  participant.
+* Each participant must retrieve an appropriate
+  [token](https://github.com/settings/tokens) for their account that allows
+  access to the `repo` scope. This token should be stored locally on their disk
+  in a file accessible to the management application.
+* Each participant should initialize their respective key: `association`,
+  `owner`, or `operator` in a secure storage solution, likely Vault. And
+  prepare another token file accessible to the manaagement application.
+* Each participant will upload their key to GitHub specifying their namespace
+  as their distinct name provided earlier:
+  * Association:
```

Comment:
Having now been able to see the end to end flow a little more clearly, I'm wondering if it might make sense to rename some of these operations. For example, renaming the *-key commands to be upload-*-key instead (or submit-*key), as they simply just upload the key to the remote storage (e.g., Github account). Likewise for "validator-config". Regarding the genesis command, it might also make sense to update it to something like "produce-genesis"? My feeling is that we probably want these commands to be as obvious as possible -- especially if they're going to be run by non-experts. (Just my 2 cents!)

Reply:
so many more characters :octocat: